### PR TITLE
Remove embedded 'cephd' code

### PR DESCRIPTION
**Pull request by dmick**
Passes make tests and ctest 

Signed-off-by: Dan Mick <dan.mick@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
retest this please

File: src/test/cli/ceph-fuse/help.t [hunk 1]
```diff
@@ -0,0 +1,25 @@
+  $ ceph-fuse --help
```
[the remark this answers is not in the document]
Comment (dmick) [reply src/test/cli/ceph-fuse/help.t]: crap, yes, good catch; branch pollution

[the remark this answers is not in the document]
Comment (dmick) [reply src/test/cli/ceph-fuse/help.t]: nope, will remove

File: src/erasure-code/CMakeLists.txt [hunk 2]
```diff
@@ -37,10 +36,3 @@ add_custom_target(erasure_code_plugins DEPENDS
     ec_lrc
     ec_jerasure
     ec_shec)
-
-if(WITH_EMBEDDED)
-  include(MergeStaticLibraries)
```
[the remark this answers is not in the document]
Comment (dmick) [reply src/erasure-code/CMakeLists.txt]: ok

**Comment by tchaikov** [comment]
```
> ceph-dashboard@0.0.0 test /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/frontend
> ng test "--browsers" "PhantomJS" "--watch=false"

[32m12 05 2018 02:55:26.745:INFO [karma]: [39mKarma v1.7.1 server started at http://0.0.0.0:9876/
[32m12 05 2018 02:55:26.750:INFO [launcher]: [39mLaunching browser PhantomJS with unlimited concurrency
[32m12 05 2018 02:55:26.770:INFO [launcher]: [39mStarting browser PhantomJS
[32m12 05 2018 02:55:57.133:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: [39mConnected on socket bG1XwTs4vMujc4TsAAAA with id 66037773
PhantomJS 2.1.1 (Linux 0.0.0): Executed 0 of 165 SUCCESS (0 secs / 0 secs)
[1A[2KPhantomJS 2.1.1 (Linux 0.0.0): Executed 1 of 165 SUCCESS (0 secs / 1.605 secs)
[1A[2KPhantomJS 2.1.1 (Linux 0.0.0) IscsiComponent should create FAILED
	Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
PhantomJS 2.1.1 (Linux 0.0.0): Executed 2 of 165 (1 FAILED) (0 secs / 7.287 secs)
PhantomJS 2.1.1 (Linux 0.0.0) IscsiComponent should create FAILED
	Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
```

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2018-05-12_08:15:46-rados-wip-kefu-testing-2018-05-12-1405-distro-basic-smithi/

**Comment by dmick** [comment]
Did you decide the dashboard test failure was unrelated (I assume)?

**Comment by tchaikov** [comment]
yeah, i believe it’s unrelated.

**Comment by smithfarm** [comment]
@dmick @tchaikov @bassam Is there a reason this should not be backported to mimic? See the discussion at https://tracker.ceph.com/issues/24437

(Since mimic is very early in its release cycle, including this now will potentially prevent loads of cherry-pick conflicts in the future.)

**Comment by dmick** [comment]
I don't know of a reason, no.  rook will not use mimic with any code that relies on cephd AFAIK.

**Pull request merged**